### DISCUSSION TITLE
如何禁止连接的箭头随着调整连接线而移动到其他锚点上？

### DISCUSSION THREAD

**Discussion by wangyexiang** [didi/LogicFlow#1584] [Q&A]
请问一下，有什么办法能够禁止箭头随连接线移动而移动，我想调整连接线的位置，而箭头需要保持到最开始连接的锚点上，而不移动。
最初：
![image](https://github.com/didi/LogicFlow/assets/23202109/3617669a-6bff-4703-b68f-490c726c5d3b)
调整后：
![image](https://github.com/didi/LogicFlow/assets/23202109/1cc8d8e1-36d7-4d94-8f63-2cc2a0153a93)

我想让箭头一直保持与左边的锚点连接，而不随调整连线的位置而移动到右边的锚点上。

**Comment by zxh666a** [comment]
请问这个问题 你找到解决办法了吗

**Comment by EdwinOlders11** [comment]
我也在苦恼这个问题，大佬有找到办法吗

**Reply by wangyexiang** [reply]
没解决

**Comment by DymoneLewis** [comment]
目前应该是没啥办法，现在边的顶点数量是固定的，拖拽不生成新的顶点，所以会跟着动
只能等后面边优化把这也优化了
